numpy/core/setup.py

```diff
@@ -685,13 +685,15 @@ def get_mathlib_info(*args):
                        join('src', 'npymath', 'npy_math_complex.c.src'),
                        join('src', 'npymath', 'halffloat.c')
                        ]
+    
+    is_msvc = customized_ccompiler().compiler_type == 'msvc'
     config.add_installed_library('npymath',
             sources=npymath_sources + [get_mathlib_info],
             install_dir='lib',
             build_info={
-                'include_dirs' : [],
-                'extra_compiler_args' : (['/GL-'] if sys.platform == 'win32' else []),
-            })  # empty list required for creating npy_math_internal.h
+                'include_dirs' : [],  # empty list required for creating npy_math_internal.h
+                'extra_compiler_args' : (['/GL-'] if is_msvc else []),
```

Comment:
This looks like a bugfix, not just cleanup, to me

Reply:
Is the flag also needed for the Intel compiler? <searches> Looks like Intel has this flag also, so we might want to disable it for that also if Python 3.5 enables it and we use that by default. @cgohlke Thoughts? Clang may be a future concern.

Reply:
Would it be okay to just add them? I am genuinely open to any solution.